starlingbank/__init__.py

```diff
@@ -4,22 +4,23 @@
 from json import dumps as json_dumps
 from base64 import b64decode
 from typing import Dict
+from datetime import datetime
 
 BASE_URL = "https://api.starlingbank.com/api/v2"
 BASE_URL_SANDBOX = "https://api-sandbox.starlingbank.com/api/v2"
 
 
+"""Build a URL from the API's base URLs."""
```

Comment:
As per [PEP257](https://peps.python.org/pep-0257/), please move doc strings back to the first line of the definition. For this and other functions below.